node/tessel-export.js

```diff
@@ -44,8 +44,8 @@ function Tessel(options) {
   }
 
   this.ports = {
-    A: options.ports.A ? new Tessel.Port('A', '/var/run/tessel/port_a', this) : null,
-    B: options.ports.B ? new Tessel.Port('B', '/var/run/tessel/port_b', this) : null,
+    A: options.ports.A ? new Tessel.Port('A', Tessel._socketPaths.A, this) : null,
+    B: options.ports.B ? new Tessel.Port('B', Tessel._socketPaths.A, this) : null,
```

Comment:
@rwaldron they are defined [here](https://github.com/tessel/t2-firmware/pull/93/files#diff-84087272b70af9498872cf6f791003abR861) but you're totally right that the port b var is wrong.

I think your third suggestion makes more sense than the way I set them up.